Microservices/event-service/src/main/java/app/sportahub/eventservice/dto/request/EventRequest.java

```diff
@@ -0,0 +1,49 @@
+package app.sportahub.eventservice.dto.request;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import jakarta.annotation.Nullable;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+
+import java.time.LocalDate;
+import java.util.List;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public record EventRequest(
+                           @NotBlank(message = "Event name must be provided")
+                           String eventName,
+
+                           @NotBlank(message = "Event type must be provided")
+                           String eventType,
+
+                           @NotBlank(message = "Sport type must be provided")
+                           String sportType,
+
+                           @NotNull
+                           LocationRequest location,
+
+                           @NotNull(message = "Date must be provided")
+                           LocalDate date,
+
+                           @NotBlank(message = "Duration must be provided")
```

Comment:
Can you make it nullable? We don't know yet if there will be durations in all and each of the events. People can choose not to specify a duration.